client/analytics/index.js

```diff
@@ -160,17 +160,17 @@ var analytics = {
 				// prevent explosion of read list metrics
 				// this is a hack - ultimately we want to report this URLs in a more generic way to 
 				// google analytics
-				if ( featureSlug.startsWith( 'read_list' ) ) {
+				if ( featureSlug.indexOf( 'read_list' ) === 0 ) {
 					featureSlug = 'read_list';
-				} else if ( featureSlug.startsWith( 'tag_' ) ) {
+				} else if ( featureSlug.indexOf( 'tag_' ) === 0 ) {
 					featureSlug = 'tag__id';
-				} else if ( featureSlug.startsWith( 'domains_add_suggestion_' ) ) {
+				} else if ( featureSlug.indexOf( 'domains_add_suggestion_' ) === 0 ) {
 					featureSlug = 'domains_add_suggestion__suggestion__domain';
-				} else if ( document.location.pathname.startsWith( '/plugins/browse/' ) ) {
+				} else if ( document.location.pathname.indexOf( '/plugins/browse/' ) === 0 ) {
 					featureSlug = 'plugins_browse__site';
-				} else if ( featureSlug.startsWith( 'read_post_feed_' ) ) {
+				} else if ( featureSlug.indexOf( 'read_post_feed_' ) === 0 ) {
 					featureSlug = 'read_post_feed__id';
-				} else if ( featureSlug.startsWith( 'read_post_id_' ) ) {
+				} else if ( featureSlug.indexOf( 'read_post_id_' ) === 0 ) {
```

Comment:
**lodash** also has a `startsWith` which could be handy here in the meantime until we get the Polyfill in because it still indicates that actual action you want to take, which `indexOf` doesn't.